resources/carrier/en/90.txt

```diff
@@ -13,26 +13,26 @@
 # limitations under the License.
 # Wiki Link : https://tr.wikipedia.org/wiki/Türkiye'deki_telefon_numaraları
 
-90501|Türk Telekom
-90505|Türk Telekom
-90506|Türk Telekom
-90507|Türk Telekom
-90551|Türk Telekom
-90552|Türk Telekom
-90553|Türk Telekom
-90554|Türk Telekom
-90555|Türk Telekom
-90556|Türk Telekom
-90559|Türk Telekom
+90501|Turk Telekom
+90505|Turk Telekom
+90506|Turk Telekom
+90507|Turk Telekom
```

Comment:
I think 9050|Turk Telekom is fine; there are no other competing prefixes, and we already validate numbers beginning with e.g. 504 as mobile, so they would otherwise be omitted.

Same below for: 9055|Turk Telekom

The wikipedia page says also somewhere else that 50x and 55x will be Turk Telekom.